### ISSUE TITLE
NTR: Twin pregnancy  (Endocrine & genitourinary)

### ISSUE THREAD

**Issue by pnrobinson** [obophenotype/human-phenotype-ontology#7959]
New term request Twin pregnancy  (elthat_130815173233):   UMLS: 0152150

**Comment by pnrobinson** [comment]
@MickeySegal

Adding new term: 
- ID: HP:6000104
- label: Twin pregnancy
- parents: [Pregnancy history (HP:0002686)](https://hpo.jax.org/app/browse/term/HP:0002686)
- pmid: PMID:33622004
- def: Maternal history of a pregancy with two fetuses in the uterus.
- comment: 
- orcid: orcid:0000-0002-0736-9199
- github ID: https://github.com/obophenotype/human-phenotype-ontology/issues/7959
- synonyms:
